spec/models/concerns/blacklight/solr/document/marc_export_override_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe Blacklight::Solr::Document::MarcExportOverride do
+  # See https://mixandgo.com/learn/how-to-test-ruby-modules-with-rspec for information on
+  # testing methods defined in modules.
+  let(:dummy_class) do
+    Class.new do
```

Comment:
Can you update the word 'dummy_class' with 'sample_class'? There was a discussion a long time ago where dummy was decided as a deprecated word. The proposed inclusive term was 'Placeholder value, sample value'. https://docs.google.com/document/d/1QXxC7hyhjUmsJyrZhkQn9rxv3Y32iIuVXzvqpMg7Ikw/edit#

Reply:
Fixed. Thank you catching this and making me aware of the document with the discussion.

Reply:
Thank you @hectorcorrea!  Maybe at some point we can go ahead and fix these terms across orangelight and bibdata.